### DISCUSSION TITLE
How do I download json-schema.org/draft/2020-12/schema and use it as local file reference in $Schema field of my custom schemas.

### DISCUSSION THREAD

**Discussion by KamalAwasthi** [json-schema-org/community#842] [Specification]
I am trying to integrate the validation in my jenkins job, but accessing and downloading the draft from an external link such as https://json-schema.org/draft/2020-12/meta/core is not allowed. I am hitting the warning while trying to run the validation. also my jenkins job keep trying for several minutes before giving failure message that it could not find schema at above link.

Is there a way I can define this draft schema in my local and then use the file link to provide in $Schema field?

**Comment by jdesrosiers** [comment]
My guess is that whatever you're using for validation doesn't support 2020-12. JSON Schema validators should have the meta-schemas for the versions they support packaged locally. They should not need to be downloaded. In the unlikely case that the validator you're using does support 2020-12 and doesn't have the meta-schemas, the validator should have some way to allow allow you to pre-load schemas. How to do that depends on the specific validator, so you'll have to look up their documentation. You can download the meta-schemas manually using the URIs in the `$id` keyword. The 2020-12 meta-schema is modular, so you'll need to download and load several schemas. Use the dialect identifier (https://json-schema.org/draft/2020-12/schema) for the main schema and download each of the schemas referenced (`$ref`) from that schema.
